openlibrary/templates/covers/add.html

```diff
@@ -63,7 +63,7 @@
                     <label id="imageWeb" for="imageUrl">$_("Or, paste in the image URL if it's on the web.")</label>
                 </div>
                 <div class="input">
-                    <input type="text" name="url" id="imageUrl" value="$data.get('url', 'https://')" onClick="if(!this._haschanged && this.value == 'https://'){this.value=''};this._haschanged=true;"/>
+                    <input type="url" name="url" id="imageUrl" value="" placeholder="https://..." />
```

Comment:
I'm not entirely sure when this can be set, but if a url is pre-specified when this template is rendered we should add it here. We can now just set the default to `''` :)

```suggestion
                    <input type="url" name="url" id="imageUrl" value="$data.get('url', '')" placeholder="https://..." />
```